code/modules/paperwork/photography.dm

```diff
@@ -32,7 +32,7 @@
 	var/icon/img		//Big photo image
 	var/scribble		//Scribble on the back.
 	var/blueprints = FALSE	//Does it include the blueprints?
-	var/info 			//Info on the camera about mobs or some shit 
+	var/info 			//Info on the camera about mobs or some shit
```

Comment:
no idea why this says the line is changed but I tried to fix it and it did nothing so w/e